packages/client/schema/ee/index.schema.js

```diff
@@ -8,74 +8,68 @@ import Dataset from 'schema/dataset.schema';
 import Jupyterhub from 'schema/jupyterhub.schema';
 import BuildImage from 'schema/ee/buildImage.schema';
 import BuildImageJob from 'schema/ee/buildImageJob.schema';
-import UsageReport from 'schema/ee/usageReport.schema';
```

Comment:
Please also delete the canner schema file.

Reply:
done